pkg/infrastructure/services/errors.go

```diff
@@ -76,6 +77,14 @@ func errorForDashboards(err error) reconciliationError {
 	}
 }
 
+func errorForServiceNotReachable(statusCode int, topicsURL string) reconciliationError {
+	return reconciliationError{
+		reason:                 v1alpha1.InternalServiceNotReachable,
+		reconciliationInterval: reconciliationIntervalAfterFetchService,
+		innerError:             fmt.Errorf("Received NOT expected status code %d while making a request to %s ", statusCode, topicsURL),
```

Comment:
It may be good to provide HTTP method information too .
Also the `topicsURL` could be renamed to something more generic, maybe `requestURL`?